scripts/feature_tests/upgrade/upgrade_common.sh

```diff
@@ -471,5 +463,14 @@ function verify_kubernetes_version_upgrade() {
             break
         fi
     done
+}
+
+function point_to_target_cluster() {
```

Comment:
The test script calls this function, right?
The test script(s) change(s) should be included in this PR.

Reply:
Indeed, the functions in here are called from other test scripts. 
The way you suggested, we have https://github.com/metal3-io/metal3-dev-env/pull/455
But, it got bigger and trying to create smaller PRs such this.

Reply:
Small PRs help the review process significantly.
Then it's good to refer to #455 as there is a dependency.